markdown/README.markdown

```diff
@@ -0,0 +1,234 @@
+Hacker's guide to UmDESK
+========================
```

Comment:
Oh, my draft Hacker's guide is here! I realize you don't use this text anymore, but I feel funny reading it now  :smiley: Please keep it in the repo!